astrobee/behaviors/inspection/scripts/stitch_panorama.py

```diff
@@ -16,133 +16,190 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+"""
+Generate/update Hugin PTO files and stitch a panorama.
+
+Example:
+  export ASTROBEE_RESOURCE_DIR=$SOURCE_PATH/astrobee/resources
+  export ASTROBEE_CONFIG_DIR=$SOURCE_PATH/astrobee/config
+  export ASTROBEE_WORLD=granite
+  export ASTROBEE_ROBOT=bsharp
```

Comment:
Having both arguments and env variables makes it counter intuitive to use the script, especially since this doesn't show up in the -h of the script. My suggestion is to use the `rospack.get_path('astrobee')` command instead and the world/robot to be an argparse (then you can use os.environ to define those for the other scripts).

Reply:
Actually, it does show up in the `-h`!

These environment variables are not used by `stitch_panorama.py` directly but rather by `undistort_image`, which uses them to find the Astrobee config files. I think they may be the same environment variables specified for ROS nodes started by the launchfile. It would probably be good to *allow* an experienced Astrobee developer to use those same variables even if we don't require it.

I'll think about a way to specify them on the command line for users who prefer not to make a bunch of `export` calls first.